### PR TITLE
create node.js 6 and 8 imagestreams for ppc64le

### PR DESCRIPTION
**Pull request by jeffdyoung**
create node.js 6 and 8 imagestreams for ppc64le

### CODE REVIEW

**Comment by jeffdyoung** [comment]
I just noticed that if I load an example template from https://raw.githubusercontent.com/jeffdyoung/nodejs-ex/ppc64le/openshift/templates/nodejs-ppc64le.json

That I get 2 nodejs entries on the UI. Should I not be using the 'builder' tag on the imagestreams?

File: imagestreams/nodejs-rhel7-ppc64le.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      {
+        "name": "latest",
+        "annotations": {
+          "openshift.io/display-name": "Node.js (Latest)",
+          "openshift.io/provider-display-name": "Red Hat, Inc.",
+          "description": "Build and run Node.js applications on RHEL 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-nodejs-container/blob/master/8/README.md.\n\nWARNING: By selecting this tag, your application will automatically update to use the latest version of Node.js available on OpenShift, including major versions updates.",
+          "iconClass": "icon-nodejs",
+          "tags": "builder,nodejs,ppc64le",
+          "supports":"nodejs",
+          "sampleRepo": "https://github.com/openshift/nodejs-ex.git"
```
Comment (liangxia) [inline imagestreams/nodejs-rhel7-ppc64le.json]: Better use https://github.com/sclorg/nodejs-ex.git to not rely on the redirect.

File: imagestreams/nodejs-rhel7-ppc64le.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        "name": "6",
+        "annotations": {
+          "openshift.io/display-name": "Node.js 6",
+          "openshift.io/provider-display-name": "Red Hat, Inc.",
+          "description": "Build and run Node.js 6 applications on RHEL 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-nodejs-container.",
+          "iconClass": "icon-nodejs",
+          "tags": "builder,nodejs,ppc64le",
+          "supports":"nodejs:6,nodejs",
+          "version": "6",
+          "sampleRepo": "https://github.com/openshift/nodejs-ex.git"
```
Comment (liangxia) [inline imagestreams/nodejs-rhel7-ppc64le.json]: Same here.

**Comment by jeffdyoung** [comment]
@liangxia I updated the power and x86 definitions. 
Are you good with keeping these changes in this PR? If not I can revert, and open a new PR in case you want to track the x86 things separate.

**Comment by liangxia** [comment]
LGTM

**Comment by hhorak** [comment]
LGTM, thanks.

**Pull request merged**